sktime/forecasting/var.py

```diff
@@ -75,6 +78,7 @@ class VAR(_StatsModelsAdapter):
         "requires-fh-in-fit": False,
         "univariate-only": False,
         "ignores-exogeneous-X": False,
```

Comment:
saym is this correct? It does not look like `X` is used inside.

I.e., this is VAR, not VAR-X or VARMAX. Right?

Reply:
while we're here, would also be great if the class docstring could say which statsmodels object it interfaces.